src/common-components/data/service.js

```diff
@@ -18,10 +18,8 @@ export async function getThirdPartyAuthContext(urlParams) {
       throw (e);
     });
   return {
-    fieldDescriptions: data.registration_fields || {},
-    optionalFields: data.optional_fields || {},
-    thirdPartyAuthContext: camelCaseObject(
-      convertKeyNames(data.context_data, { fullname: 'name' }),
-    ),
+    fieldDescriptions: data.registrationFields,
+    optionalFields: data.optionalFields,
```

Comment:
```suggestion
    fieldDescriptions: data.registrationFields || {},
    optionalFields: data.optionalFields || {},
```

Reply:
I removed these conditions because they were causing a codecov issue. In order to address the issue, those lines need to be added back in and the tests will also need to be updated accordingly.

Reply:
i don't understand this. could you please elaborate more ?